tests/filecheck/dialects/scf/for_iv.mlir

```diff
@@ -3,7 +3,7 @@
   %lb = "arith.constant"() {"value" = 0 : index} : () -> index
   %ub = "arith.constant"() {"value" = 42 : index} : () -> index
   %step = "arith.constant"() {"value" = 7 : index} : () -> index
-// CHECK: Wrong number of block arguments, expected 1, got 0. The body must have the induction variable and loop-carried variables as arguments.
+// CHECK: Body block must at least have induction var as block arg
```

Comment:
this reads weird

Reply:
How about this:

```suggestion
// CHECK: Body block must have induction var as first block arg
```

Reply:
It was what `mlir-opt` was printing as error message; not married to it :)

Reply:
Ah maybe in that case we can stick with mlir-opt, your call